kolibri/plugins/epub_viewer/assets/src/views/SearchSideBar.vue

```diff
@@ -62,7 +62,7 @@
     </transition>
 
     <ol
-      v-show="searchHasBeenMade && !searchIsLoading && searchResults.length > 0"
+      v-if="searchHasBeenMade && !searchIsLoading && searchResults.length > 0"
       ref="searchResultsList"
       class="search-results-list"
     >
```

Comment:
One thought that occurs is that we could simply do the highlight styling ourselves, and pass in the styled text into the `KButton` slot, rather than passing it as text and then doing the post hoc highlighting with Mark.

Something like:
```
        <KButton
          :text="item.excerpt"
          appearance="basic-link"
          class="search-results-list-item-button"
          @click="$emit('navigateToSearchResult', item)"
        >
        <span v-for="i, text in item.excerpt.split(searchQuery)" :key="i" :class="text.toLowerCase() === searchQuery.toLowerCase() ? 'highlight' : ''>
           {{ text }}
       </span>
    </KButton>
```

Obviously you would need to make the 'split' a bit more nuanced, as we are doing case insensitive searches, so you'd probably need a method to handle this better. And would need to define a `highlight` class to handle the highlighting.

Reply:
Yes, I think it's a super good suggestion, it could be added to #10008 since precisely in that PR I compute the `before the match`, `match` and `after the match` text portions in separate spans, and that preprocessing is calculated in another method that has in account the case insensitive.

Reply:
Great, I'll leave that to you to add in #10008!